reference/CloudPay-API-Specification.yaml

```diff
@@ -2261,6 +2261,277 @@ paths:
+                            Name:
+                              type: string
+                              description: The description of the custom metadata field
+                              readOnly: true
+                            Code:
+                              type: string
+                              description: The code identifier of the metadata field
+                              readOnly: true
+                            Value:
+                              type: string
+                              description: The value of the custom metadata
+                              readOnly: true
+                          required:
+                            - Name
+                            - Code
+                            - Value
+                    required:
+                      - Id
+                      - Title
+                      - Body
+                      - Amount
+                      - Currency
+                      - AmountFormatted
+                      - HasFreeTrial
+                      - PurchaseUrl
+                      - Interval
+                      - IAPData
+                      - Metadata
+              examples:
+                Package:
+                  value:
+                    Id: 82fa21ed-6e23-469f-b802-4e2a67679c54
+                    Title: Rest of the world - Monthly Subscription
+                    Body: '<p class="package-body">Full Coverage of TV tournaments, Players Championships &amp; European Tour events, interviews and archive content</p><ul><li>European Tour Streaming</li><li>Players Championship Streaming</li><li>TV Events (Live)</li><li>TV Events (Highlights)</li><li>Archive &amp; Feature Content</li><li>Access to the PDC Newsletter</li><li>Partner discounts</li></ul>'
+                    Style: null
+                    Amount: 9.99
+                    Currency: GBP
+                    AmountFormatted: £9.99
+                    HasFreeTrial: false
+                    TrialDuration: null
+                    PurchaseUrl: "https://payments.streamamg.com/sso/startbasket/82fa21ed-6e23-469f-b802-4e2a67679c54?apijwttoken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c&lang=en"
+                    Interval: 1months
+                    Duration: 1months
+                    IAPData: []
+                    Metadata:
+                      - Name: packageType
+                        Code: packageTypeCode
+                        Value: video
+        "400":
+          description: "Bad Request"
+          content:
+            application/json:
+              schema:
+                description: ""
+                type: object
+                x-examples:
+                  BadRequestError:
+                    statusCode: 400
+                    message: "Package should be updated."
+                properties:
```

Comment:
Package should be updated on a 400?

Reply:
yes this 'Package should be updated.' error occurs only when price for the package doesn't exist, so the user has to update the package with the proper price in order to be able to retrieve it

do you think maybe 403 fits better in here?

Reply:
Nope 400 description best matches the error